data/json/mapgen/dumpsite.json

```diff
@@ -15,56 +15,56 @@
     },
     "items": {
       ".": [
```

Comment:
Various bits like this and the custom-flags one want booting into an item group which do support the fields (others are less important ig)

Reply:
This block in particular is a clean revert of #80224, should it be commented out instead? I don't want to get involved with itemgroups.